packages/docs/community/release-process/smart-contracts.md

```diff
@@ -72,11 +72,23 @@ yarn verify-deployed -n $NETWORK -b $PREVIOUS_RELEASE -f
 
 ### Check Backward Compatibility
 
-Use the following script to compile candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
+This script performs some automatic checks to ensure that smart contract
+versions have been updated correctly.
+
+Specifically, it compiles candidate and current release contracts to check storage layout and ABI backwards compatibility, subject to the following exceptions:
   1. If the STORAGE version has changed, does not perform backwards compatibility checks
   2. If the MAJOR version has changed, checks storage layout compatibility but not ABI compatibility
 
-For changed contracts, checks that new version number is strictly greater than previous release. For unchanged contracts, checks that version number exactly matches previous release. Outputs a JSON backwards compatibility report.
+Versions being updated correctly means that no backwards incompatibilities were
```

Comment:
Requiring that the correct version numbers are updated in source code whenever changes are made ensures that no backwards incompatibilities are accidentally introduced.